flixel/input/gamepad/XInputButtonID.hx

```diff
@@ -0,0 +1,40 @@
+package flixel.input.gamepad;
+
+import flixel.input.gamepad.FlxGamepad;
+
+/**
+ * Button IDs for generic XInput controllers
+ */
+class XInputButtonID
+{
+	/**
+	* Button IDs
+	*/
+	public static inline var A:Int = 6;
+	public static inline var B:Int = 7;
+	public static inline var X:Int = 8;
+	public static inline var Y:Int = 9;
+	public static inline var LB:Int = 15;
+	public static inline var RB:Int = 16;
+	public static inline var BACK:Int = 10;
+	public static inline var START:Int = 12;
+	public static inline var LEFT_STICK_BTN:Int = 13;
+	public static inline var RIGHT_STICK_BTN:Int = 14;
+	
+	public static inline var GUIDE:Int = 11;
```

Comment:
In the LIME gamepad API, it seems to stand in for the "Xbox", "PS", and/or "Home" button consoles stick in the middle these days.